app/views/contact/new.html.erb

```diff
@@ -1,16 +1,14 @@
-<% title "Contact Us" %>
+<% title 'Contact Us' %>
 
-<%= simple_form_for @message, :url => url_for( :action => "new", :controller => "contact"), :html => { :class => "form-horizontal" } do |f| %>
+<%= simple_form_for @message, url: url_for(action: 'new', controller: 'contact'), html: { class: 'form-horizontal' } do |f| %>
 
-<%= f.error_notification %>
+  <%= f.error_notification %>
 
-<%= f.input :name %>
-<%= f.input :email %>
-<%= f.input :subject %>
-<%= f.input :body, :as => :text, :input_html => { :class => "span6" } %>
+  <%= f.input :name %>
+  <%= f.input :email %>
+  <%= f.input :subject %>
+  <%= f.input :body, as: :text, input_html: { class: 'form-control' } %>
 
-<div class="form-actions">
-  <%= f.button :submit, "Send Message" %>
-</div>
+  <%= f.button :submit, 'Send Message' %>
```

Comment:
Have you restarted the server since pulling? Since it was a change in code that's only loaded on server start, you would still get the same error until you restart the server.